components/prism-xeora.js

```diff
@@ -0,0 +1,114 @@
+(function(Prism) {
+	Prism.languages.xeora = Prism.languages.extend('markup', {
+		'constant': {
+			pattern: /\$(?:DomainContents|PageRenderDuration)\$/,
+			inside: {
+				'punctuation': {
+					pattern: /\$/
+				}
+			}
+		},
+		'variable': {
+			pattern: /\$(@?)(#+|[\-+*~=^])?[\w\.]+\$/,
+			inside: {
+				'punctuation': {
+					pattern: /\$|\./
+				},
+				'operator': {
+					pattern: /@|#+|\^|-|\+|\*|~|=/
+				}
+			}
+		},
+		'function-inline': {
+			pattern: /\$F:[\.\w\-]+\?[.\w\-]+(,((\|)?([#\.^\-+*~]*([\w+][^$]*)|=([\S+][^$]*)|@[#\-]*(\w+\.)[\w+\.]+)?)*)?\$/,
+			inside: {
+				'variable': {
+					pattern: /(,|\|)(@?)(#+|[\-+*~=^])?[\w\.]+/,
+					inside: {
+						'punctuation': {
+							pattern: /\.|,|\|/
+						},
+						'operator': {
+							pattern: /@|#+|\^|-|\+|\*|~|=/
+						}
+					}
+				},
+				'punctuation': {
+					pattern: /\$|:|\?|\.|,|\|/
+				}
+			},
+			alias: 'function'
+		},
+		'function-block': {
+			pattern: /\$XF:{[\.\w\-]+\?[\.\w\-]+(,((\|)?([#\.^\-+*~]*([\w+][^$]*)|=([\S+][^$]*)|@[#\-]*(\w+\.)[\w+\.]+)?)*)?}:XF\$/,
+			inside: {
+				'punctuation': {
+					pattern: /\$|:|{|}|\?|\.|,|\|/
+				}
+			},
+			alias: 'function'
+		},
+		'directive-inline': {
+			pattern: /\$\w(#\d+(\+)?)?(\[[\.\w\-]+])?:[\/\.\w\-]+\$/,
+			inside: {
+				'punctuation': {
+					pattern: /\$\w:|\$C\[|\$C#\d|\$|:|{|\[|]/,
+					inside: {
+						'tag': {
+							pattern: /#\d/
+						}
+					}
+				}
+			},
+			alias: 'function'
+		},
+		'directive-block-open': {
+			pattern: /\$\w+:{|\$\w(#\d+(\+)?)?(\[[\.\w\-]+])?:[\.\w\-]+:{(![A-Z]+)?/,
```

Comment:
You can remove the parentheses around `(\+)?`
You can remove the escapes in `[\.\w\-]` -> `[.\w-]`
Please make the groups non-capturing.